app-media-image-capture.html

```diff
@@ -391,20 +411,91 @@
           });
         },
 
-        __updateImageCaptureOptions: function() {
-          this.debounce('updateImageCaptureOptions', function() {
-            if (this.imageCapture == null || this.photoSettings == null) {
-              return;
+        __updateTrackConstraints: function() {
```

Comment:
Sooooo I think the plight of iron-selector that we had a gigantor multi prop observer but instead of using the arguments, we were using the `this.` properties and this is exactly what it's doing :(

Sorry I didn't catch this at the last review